app/controllers/application_controller.rb

```diff
@@ -90,7 +90,17 @@ def check_accounts_enabled
 
   helper_method :accounts_enabled?
   def accounts_enabled?
-    Rails.configuration.feature_flag_govuk_accounts
+    return false unless Rails.configuration.feature_flag_govuk_accounts
+
+    if @check_accounts_enabled.nil?
+      @check_accounts_enabled = true
+      begin
+        RestClient.get(Services.accounts_api)
+      rescue RestClient::ServiceUnavailable
+        @check_accounts_enabled = false
+      end
```

Comment:
Hmm, I guess this will also throw other errors too, I think we'll also need a `rescue StandardError` clause which doesn't do anything, and it'd be good to have a test that some other 5xx error (like just a plain 500) results in `accounts_enabled? == true`

Reply:
Righto that makes sense, I'll do that now!

Reply:
https://github.com/alphagov/finder-frontend/pull/2369/commits/a94264ac25243f699520efe6b4931e43e0f9c604